app/controllers/followships_controller.rb

```diff
@@ -5,6 +5,8 @@ def create
       respond_to do |format|
         format.js
       end
+      @notification=Notification.new(user_id:@followship.followee_id,actor_id:@followship.user_id,notification_type:params[:controller],target_type:"follow",target_id:@followship.followee_id)
```

Comment:
Make a method in Notification Model and move this code there.

Reply:
Done